Assets/Plugins/GameJolt/Scripts/UI/Manager.cs

```diff
@@ -70,12 +71,14 @@ public void ShowSignIn()
 
 		/// <summary>
 		/// Shows the sign in windows.
+		/// If the user's credentials are stored in PlayerPrefs, it will populate the fields with them.
 		/// </summary>
 		/// <param name="signedInCallback">A callback function accepting a single parameter, a boolean indicating whether the user has been signed-in successfully.</param>
 		/// <param name="userFetchedCallback">A callback function accepting a single parameter, a boolean indicating whether the user's information have been fetched successfully.</param>
-		public void ShowSignIn(Action<bool> signedInCallback, Action<bool> userFetchedCallback = null)
+		/// <param name="autoLogin">If true the user will be automatically logged in with the stored credentials.</param>
+		public void ShowSignIn(Action<bool> signedInCallback, Action<bool> userFetchedCallback = null, bool autoLogin = false)
```

Comment:
I don't think `autoLogin` should be an option  for the developer to set. It should rather be simply dependent on the fact that there are some stored credentials.

Furthermore, I think this is actually the wrong place to do so. There is already some auto login for web based games (because the GameJolt website passes the credential to Javascript) so IMO, the auto login from stored credential should happen at the same place. What do you think?

Reply:
sounds good to me, I will look into it